src/types/label.js

```diff
@@ -1,21 +1,18 @@
-import {drawBox, drawLabel, measureLabelSize, isLabelVisible, getChartPoint, getRectCenterPoint, toPosition, setBorderStyle, getSize, inBoxRange, isBoundToPoint, getChartRect, getRelativePosition} from '../helpers';
+import {drawBox, drawLabel, measureLabelSize, getChartPoint, getRectCenterPoint, toPosition, setBorderStyle, getSize, inBoxRange, isBoundToPoint, getChartRect, getRelativePosition} from '../helpers';
 import {color, toPadding} from 'chart.js/helpers';
 import {Element} from 'chart.js';
 
 export default class LabelAnnotation extends Element {
 
   inRange(mouseX, mouseY, useFinalPosition) {
-    return this.visible && inBoxRange(mouseX, mouseY, this.getProps(['x', 'y', 'width', 'height'], useFinalPosition));
+    return this.options.display && inBoxRange(mouseX, mouseY, this.getProps(['x', 'y', 'width', 'height'], useFinalPosition));
   }
 
   getCenterPoint(useFinalPosition) {
     return getRectCenterPoint(this.getProps(['x', 'y', 'width', 'height'], useFinalPosition));
   }
 
   draw(ctx) {
-    if (!this.visible) {
```

Comment:
Not sure how it works if the label has an empty content. I think drawCallout will be invoked.

Reply:
AFAIK a label is displayable if display/enabled  is not undefined and the content is consistent.

Reply:
So is the "empty content case" for hiding a configuration error or is it another way of hiding label annotaton?
if the latter, the use case can be achieved with scriptable `enabled`:
`enabled: (ctx, opts) => !!opts.content`

Reply:
restored the empty content behavior

Reply:
> So is the "empty content case" for hiding a configuration error or is it another way of hiding label annotation?

I think, since version 0.5.7, this was implemented (in line label) to avoid to perform any "label" calculation. In the label annotation we used the same concept.  I don't know why was implemented in this way but I think for both the reasons.
I recall that at that time I thought that `enabled` option was useless because you could check only the content.